app/views/user_mailer/already_confirmed.html.slim

```diff
@@ -0,0 +1,14 @@
+html
+  head
+    meta content='text/html; charset=UTF-8' http-equiv='Content-Type'
+  body
+
+    p= 'A request was made to resend confirmation instructions to this email.' \
```

Comment:
In most other slim templates the style seems to be:

```
p
  'foo bar
  'baz
```

(I think the = is not needed for pure strings.)